deployment/README.md

```diff
@@ -304,9 +250,8 @@ Then create a DNS record based on that ADDRESS value in your DNS registrar.
 [iam-auth]: https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html
 [ingress-controller]: https://github.com/kubernetes/ingress-nginx
 [ingress-def]: https://kubernetes.io/docs/concepts/services-networking/ingress/
-[k8s-terraform]: https://github.com/FuelLabs/fuel-core/tree/master/deployment/terraform
 [kubectl-cli]: https://kubernetes.io/docs/tasks/tools/
-[monitoring-deploy]: https://github.com/FuelLabs/fuel-core/blob/master/deployment/scripts/monitoring-deploy.sh
-[monitoring-ingress-deploy]: https://github.com/FuelLabs/fuel-core/blob/master/deployment/scripts/monitoring-ingress-deploy.sh
+[monitoring-deploy]: https://github.com/FuelLabs/fuel-core/blob/master/deployment/scripts/fuel-core-ingress-deploy.sh
+[monitoring-ingress-deploy]: https://github.com/FuelLabs/fuel-core/blob/master/deployment/scripts/fuel-core-ingress-deploy.sh
```

Comment:
monitoring should reside in the infrastructure repo, not in fuel-core

Reply:
included in current infrastructure PR "#31" on the board- set it up on create-k8s.sh script